specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -6773,6 +6806,20 @@
       },
       "description": "The instance view of a dedicated host."
     },
+    "DedicatedHostInstanceViewWithName": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the dedicated host."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
```

Comment:
Is this correct?  Resource contains id, name, type, location and tag.

Reply:
Will update to #/definitions/DedicatedHostInstanceView. DedicatedHostInstanceViewWithName has everything that DedicatedHostInstanceView has, plus "name".